README.md

```diff
@@ -78,21 +78,25 @@ package versions <1.0.0 follow
 
 ## Badge
 
-[![Built with the Deno Standard Library](./badge.svg)](https://jsr.io/@std)
+> [!NOTE]
+> Previously, this repo hosted the badge SVG file. Now, the badge is retrieved
+> directly from [Shields.io](https://shields.io/).
+
+[![Built with the Deno Standard Library](https://img.shields.io/badge/Built_with_deno__std-blue?logo=deno)](https://jsr.io/@std)
```

Comment:
Do we still use `deno_std` name? Or should we rename it to `std`?

Reply:
`std` seems fine to me